testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -220,6 +219,16 @@ def test_load_new(self):
         u.load_new(PDB_small)
         assert_equal(len(u.trajectory), 1, "Failed to load_new(PDB)")
 
+    def test_load_new_returns_Universe(self):
+        u = mda.Universe(PSF)
+        result = u.load_new(PDB_small)
+        assert result is u
+
+    def test_load_new_None_returns_Universe(self):
+        u = mda.Universe(PSF)
+        result = u.load_new(None)
```

Comment:
I just looked at the code again https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/core/universe.py#L276 and I have to say, it keeps that code cleaner if we allow `load_new(None)`. And it makes sense semantically, too. For a functional approach it is also nicer if you don't have to catch exceptions in a chain of methods but instead process inputs in a meaningful way.

I'd actually keep the current behavior.